src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -567,18 +567,18 @@ def launch_fluent(
                 kwargs.update(cwd=cwd)
             if topy:
                 if isinstance(topy, str):
-                    journal_name = topy.split(".")[0]
-                    journal_ext = topy.split(".")[1]
-                    launch_string += f' -i {journal_name}.{journal_ext} -command="(api-start-python-journal \\\"\\\"{journal_name}.py\\\"\\\")"'  # noqa: E501
+                    name = topy.split(".")[0]
+                    ext = topy.split(".")[1]
```

Comment:
or something like:
```
name, ext = topy.split(".")[:2]
```
because of the duplication
?

Reply:
Done. Thank you.